src/test/java/jnr/ffi/PlatformTest.java

```diff
@@ -0,0 +1,106 @@
+package jnr.ffi;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+
+public class PlatformTest
+{
```

Comment:
Our convention is brace-inline, not brace-following. The rest of the file is brace-following also.